### ISSUE TITLE
readme request

### ISSUE THREAD

**Issue by learnifysean** [yusuketomoto/chainer-fast-neuralstyle#8]
what does the parameters mean? 

lambda_tv,lambda_feat,lambda_style, lr...etc

how do they influence the outcome of trained images?

**Comment by bucktoothsir** [comment]
lr is learning rate.
lambda_style and lambda_feat are the weighting factors for style and content respectively.
lambda_tv is the total variation factors.
I think you should read the paper " Perceptual Losses for Real-Time Style Transfer and Super-Resolution" firstly.
all of the parameters are explained in it.

**Comment by NorthFoxz** [comment]
@bucktoothsir thanks! But after reading the material, i still don't understand how to emphasis the color of the original image?

**Comment by austingg** [comment]
@bucktoothsir  color is somewhat low level feature, if you only want to enchance the color, you can change the weight of style feature, maybe you should dive into the code.
